cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -592,9 +595,16 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
                                 .build();
 
                         CertificateAuthority clusterCaConfig = kafkaAssembly.getSpec().getClusterCa();
-
                         // When we are not supposed to generate the CA but it does not exist, we should just throw an error
                         checkCustomCaSecret(clusterCaConfig, clusterCaCertSecret, clusterCaKeySecret, "Cluster CA");
+                        KafkaClusterTemplate kafkaClusterTemplate = kafkaAssembly.getSpec().getKafka().getTemplate();
+                        Map<String,String> ClusterCaCertLabels = null;
+                        Map<String,String> ClusterCaCertAnnotations = null;
```

Comment:
I think these should start with lowercase (`clusterCaCertLabels ` and `clusterCaCertAnnotations `)